Makefile

```diff
@@ -25,7 +25,7 @@ ifneq ($(findstring onnx,$(TARGETS)),onnx)
     PYTEST_ARGS := $(PYTEST_ARGS) --ignore tests/sparseml/onnx
 endif
 ifneq ($(findstring pytorch,$(TARGETS)),pytorch)
-    PYTEST_ARGS := $(PYTEST_ARGS) --ignore tests/sparseml/pytorch
+    PYTEST_ARGS := $(PYTEST_ARGS) --ignore tests/sparseml/pytorch --ignore tests/sparseml/modifiers
```

Comment:
We should think about a better way to separate out the pytorch tests, because eventually we will have other frameworks in this folder. This will also skip any framework-independent base tests we write

Reply:
Updated to use a regex for files inside sparseml/modifiers